### ISSUE TITLE
Add pagination to changelog

### ISSUE THREAD

**Issue by jhg03a** [apache/trafficcontrol#3530]
As an SRE, it's sometimes useful to be able to see what happened more than 1024 events ago in TO.  Certain larger scale operations can cause that to only go back less than a day.  The data still exists in the TODB if a query is manually issued directly inside the database, but this is both painful and dangerous.

**Comment by mitchell852** [comment]
this is somewhat related to #2915 but i think this solution is better fetch the results from the server as you paginate thru.

**Comment by jhg03a** [comment]
Correct.  Not asking for new ceilings or caps trying to load all the data at once.
